index.rst

```diff
@@ -956,6 +956,17 @@ Additional Resources
    beginner/vt_tutorial
    intermediate/tiatoolbox_tutorial
 
+.. toctree::
+   :maxdepth: 1
+   :includehidden:
+   :hidden:
+   :caption: Torch Compiler
+
+   intermediate/torch_compile_tutorial
+   intermediate/compiled_autograd_tutorial
+   intermediate/transformer_building_blocks
```

Comment:
This is not a torch.compile tutorial (https://pytorch.org/tutorials/intermediate/transformer_building_blocks.html). It uses torch.compile, but it is more about optimizing transformers.